CONTRIBUTING.md

```diff
@@ -0,0 +1,173 @@
+# Framework Philosophies
+
+This document outlines the goals, ideas, and roadmap for the rg-bot library (as well as our public-facing libraries in general). This is a living document, and we encourage players to participate and contribute to this document.
+
+## Core Facts
+
+The success of this library is dictated by various core facts that should be true. Our goal is to make sure that these facts are always true, and any change that causes one of these facts to not be true should be re-evaluated.
+
+Note that some of these facts require testing the library against real users. We don't need to evaluate these specific items every time a change occurs in the library, but we should make that evaluation often.
```

Comment:
I'm not sure what you're getting at here from a Contributor perspective. Is this intended to state that when validation against our facts requires access to a production environment that we (the RG team) will be responsible for that validation? It sounds more like you're stating that not every change needs to be evaluated against our core facts, but that defeats the purpose of having them so I'm a bit confused on this one.

Reply:
By this suggestion, I meant that it is probably not feasible to run these surveys with every change we make, so I wanted to give some leeway so that we can use our best judgment, and batch multiple changes into one survey. Thoughts?

Reply:
That's reasonable - gathering feedback from users will definitely take time. With this in mind, I'd suggest rewording to something like...

"Some of the following core facts require gathering feedback from real users, which is not feasible as a requirement for accepting every change to the library as it is proposed. Therefore, it is the responsibility of the Regression Games team to evaluate compliance with these facts at its discretion on a regular basis."